components/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/DiscoveryUtil.java

```diff
@@ -42,4 +43,18 @@ public static boolean isUseEntityIdAsIssuerInOidcDiscovery() {
         }
         return Boolean.parseBoolean(useEntityIdAsIssuerInDiscovery);
     }
+
+    /**
+     * Check whether mutual TLS aliases are enabled. This is used to determine whether to display the mutual TLS
+     * endpoints in the well-known endpoint.
+     * @return true if mutual TLS aliases are enabled.
+     */
+    public static boolean isMutualTLSAliasesEnabled() {
+
+        String mutualTLSAliasesEnabled = IdentityUtil.getProperty(MUTUAL_TLS_ALIASES_ENABLED);
+        if (StringUtils.isEmpty(mutualTLSAliasesEnabled)) {
```

Comment:
isn't this condition can cover from Boolean.parseBoolean itself?

Reply:
fixed with [a46ca3c](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2440/commits/a46ca3cd87115788e87d85cf34db82de1eefea53)